packages/@lwc/synthetic-shadow/src/faux-shadow/shadow-token.ts

```diff
@@ -47,16 +52,14 @@ defineProperty(Element.prototype, KEY__SHADOW_TOKEN, {
     configurable: true,
 });
 
-function recursivelySetShadowResolver(node: Node, fn: any) {
-    (node as any)[KEY__SHADOW_RESOLVER] = fn;
```

Comment:
Note we used to set this prop on the root as well, but it was useless because it was already set. (This is implied by line 67 in the old code.)